source/key-concepts/variables.md

```diff
@@ -1,42 +1,42 @@
 # Variables and formulas
 
-A variable is property of a person, or an entity (e.g. a family).
+A **variable** is property of a _person_, or an _entity_ (e.g. a _family_).
```

Comment:
> Connected to `source/key-concepts/simulation.md`, `# Computing variables`; the main idea here is to keep in the documentation the information that a variable comes from the legislation:
```suggestion
The legislation of a country contain entities characteristics and computable rules. Entities characteristics are modeled into **input variables**. Computable rules can be modeled into **variables with formulas**.
A **variable** is property of a _person_, or an _entity_ (e.g. a _family_).
```

Reply:
Done as requested with old and more British spelling of modelled.